js/quote-generator.js

```diff
@@ -0,0 +1,55 @@
+const randomBtn = document.querySelector("#random-btn");
+const allAuthorsBtn = document.querySelector("#all-authors-btn");
+const quote = document.querySelector(".quote");
+let quoteAuthor = document.querySelector(".quote-author");
+let quoteGenre = document.querySelector(".quote-genre");
+const allAuthorsContainer = document.querySelector(".all-authors-container");
+const gridContainer = document.querySelector(".grid-container");
+const topButtons = document.querySelector(".top-buttons");
+const footer = document.querySelector("footer");
+
+async function getRandomQuote() {
```

Comment:
This fn is doing a lot of things

It gets a quote, it sets a quote

Separate things of getting some data and setting some data. This is [called](https://en.wikipedia.org/wiki/Command%E2%80%93query_separation#:~:text=Command%2Dquery%20separation%20(CQS),the%20caller%2C%20but%20not%20both.) "Command Query Separation" 

One function fetched quote, another sets the quote to a certain dom element